From 6f8169421d619f7cf764f4dc0a0b6a16a3201c3b Mon Sep 17 00:00:00 2001
From: Marcus Lundblad <ml@update.uu.se>
Date: Wed, 22 Jun 2016 23:30:52 +0200
Subject: [PATCH] layerPopover: Remove ability to switch b/w street and aerial

As the aerial tiles currently are provided by MapQuest, the switcher
buttons is not needed currently.
Keeps the ability to load custom map layers.

https://bugzilla.gnome.org/show_bug.cgi?id=764841
---
 data/ui/layers-popover.ui | 48 ++---------------------------------------------
 src/layersPopover.js      |  6 +-----
 2 files changed, 3 insertions(+), 51 deletions(-)

diff --git a/data/ui/layers-popover.ui b/data/ui/layers-popover.ui
index a33f17d..cd5b71e 100644
--- data/ui/layers-popover.ui
+++ data/ui/layers-popover.ui
@@ -10,50 +10,6 @@
         <property name="valign">center</property>
         <property name="row_spacing">5</property>
         <child>
-          <object class="GtkRadioButton" id="streetLayerButton">
-            <property name="visible">True</property>
-            <property name="can-focus">True</property>
-            <property name="action-name">win.map-type</property>
-            <property name="action-target">"STREET"</property>
-            <property name="draw-indicator">False</property>
-            <style>
-              <class name="layer-radio-button"/>
-            </style>
-            <child>
-              <object class="GtkImage" id="streetLayerImage">
-                <property name="visible">True</property>
-                <property name="resource">/org/gnome/Maps/ui/maptype-street.png</property>
-              </object>
-            </child>
-          </object>
-          <packing>
-            <property name="left-attach">0</property>
-            <property name="top-attach">0</property>
-          </packing>
-        </child>
-        <child>
-          <object class="GtkRadioButton" id="aerialLayerButton">
-            <property name="visible">True</property>
-            <property name="can-focus">True</property>
-            <property name="action-name">win.map-type</property>
-            <property name="action-target">"AERIAL"</property>
-            <property name="draw-indicator">False</property>
-            <style>
-              <class name="layer-radio-button"/>
-            </style>
-            <child>
-              <object class="GtkImage" id="aerialLayerImage">
-                <property name="visible">True</property>
-                <property name="resource">/org/gnome/Maps/ui/maptype-aerial.png</property>
-              </object>
-            </child>
-          </object>
-          <packing>
-            <property name="left-attach">0</property>
-            <property name="top-attach">1</property>
-          </packing>
-        </child>
-        <child>
           <object class="GtkListBox" id="layersListBox">
             <property name="name">layers-list-box</property>
             <property name="visible">false</property>
@@ -65,7 +21,7 @@
           </object>
           <packing>
             <property name="left-attach">0</property>
-            <property name="top-attach">2</property>
+            <property name="top-attach">0</property>
           </packing>
         </child>
         <child>
@@ -76,7 +32,7 @@
           </object>
           <packing>
             <property name="left-attach">0</property>
-            <property name="top-attach">3</property>
+            <property name="top-attach">1</property>
           </packing>
         </child>
       </object>
diff --git a/src/layersPopover.js b/src/layersPopover.js
index f37109f..ab20760 100644
--- src/layersPopover.js
+++ src/layersPopover.js
@@ -78,9 +78,7 @@ const LayersPopover = new Lang.Class({
     Name: 'LayersPopover',
     Extends: Gtk.Popover,
     Template: 'resource:///org/gnome/Maps/ui/layers-popover.ui',
-    InternalChildren: [ 'streetLayerButton',
-                        'aerialLayerButton',
-                        'layersListBox',
+    InternalChildren: [ 'layersListBox',
                         'loadLayerButton' ],
 
     _init: function(params) {
@@ -92,8 +90,6 @@ const LayersPopover = new Lang.Class({
                       transitions_enabled: false,
                       visible: false });
 
-        this._aerialLayerButton.join_group(this._streetLayerButton);
-
         this.get_style_context().add_class('maps-popover');
 
         this._layersListBox.bind_model(this._mapView.shapeLayerStore,
-- 
2.8.1
